Authentication/Validators/TextInputValidatorType.swift

```diff
@@ -14,7 +14,7 @@ import Foundation
  */
 public enum ValidationResult {
     
-    public static func invalid(errorMessage: String) -> ValidationResult {
+    public static func invalid(_ errorMessage: String) -> ValidationResult {
```

Comment:
I ended up removing this method, because I renamed the enum cases to lowercase to follow naming conventions and it conflicted with this function and the following one. I think everything still works fine. You can check in the new commit the exact change I made.